dashboard/src/t5gweb/ui.py

```diff
@@ -301,6 +301,7 @@ def trends():
         new_comments=get_trending_cards(cards),
         jira_server=cfg["server"],
         page_title="trends",
+        sla_settings=cfg["sla_settings"],
```

Comment:
Do we need to add sla_settings for this view? Don't think it's used in that template (unless there are future plans)